tests/acceptance/acceptance-test.js

```diff
@@ -0,0 +1,174 @@
+import Ember from 'ember';
```

Comment:
What coverage is this providing that our existing tests don't? Is this basically testing the test helpers?